docs/docs/spark-configuration.md

```diff
@@ -190,6 +190,7 @@ df.write
 | compression-codec      | Table write.(fileformat).compression-codec | Overrides this table's compression codec for this write      |
 | compression-level      | Table write.(fileformat).compression-level | Overrides this table's compression level for Parquet and Avro tables for this write |
 | compression-strategy   | Table write.orc.compression-strategy       | Overrides this table's compression strategy for ORC tables for this write |
+| distribution-mode | `Range` if Sort Order Defined ; `Hash` if Partition Defined; `None` otherwise | Override this table's distribution mode for this write |
```

Comment:
```suggestion
| distribution-mode | `Range` if sort order is defined ; `Hash` if partition is defined; `None` otherwise | Override this table's distribution mode for this write |
```